src/cpp/session/modules/SessionFiles.R

```diff
@@ -149,6 +149,16 @@ for (binding in bindings)
    .rs.scalar(identical(file.info(path)$isdir, FALSE))
 })
 
+.rs.addJsonRpcHandler("file_exists", function(path)
```

Comment:
Feels like this needs a better name, since it's not testing for existence anymore; it's producing an aliased path.